src/pages/uxp/versions.md

```diff
@@ -8,17 +8,17 @@ index_desc: General information on UXP and its dependencies
 
 ### Applications Integrated with UXP
 
-| Application | UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
-| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
-|Photoshop|	24.1 |	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
-|InDesign|	- |	18.0 |	 17.4	|-|-| 17.1 | 17.0|
-|InDesign Server|	- |	18.0 | 17.4	|-|-| 17.1 | 17.0|
-|XD|	- |	55 | 54	|-|-|53| 45 |
-|InCopy|	- |	18.0 |	 17.4	|-|-|17.1| 17.0|
-|Illustrator|	- |	27.0|-| 26.4 | 26.3 | 23.2 | 23.0|
-|Premiere Pro|	- | 23.0	| 22.6 | 22.5 || 22.3 | 22.0 |
-|After Effects|	- |	23.0| 22.6 | 22.5 || 22.3 | 22.0 |
-|Animate |	- | -| 23.0 |-|-|- |22.0|
-|Bridge     |	- | 13.0 |-|-|-|-| 12.0|
-|Rush       |	- | |||| ||
-|Aero       |	- | |||| ||
+| Application | UXP v6.5| UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
+| ------------- |  ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+|Photoshop|	24.1 |	24.1 |	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
+|InDesign|	18.1 |	- |	18.0 |	 17.4	|-|-| 17.1 | 17.0|
```

Comment:
@kareemmyk  Can you verify the row that has Indesign information?

Reply:
@padmkris123 .... 
it should be ```-``` for UXP 6.5 as well. Did you receive any confirmation on the version of InDesign?
This is what I have published
| Application | UXP v6.5| UXP v6.4| UXP v6.3| UXP v6.2 | UXP v6.1 | UXP v6.0 | UXP v5.6 | UXP v5.5 |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
|Photoshop|24.1|24.1|	24.0 | 23.5 | 23.4 | 23.3 | 23.2 | 23.0 |
|InDesign|	-|	-|	18.0 |	 17.4	|-|-| 17.1 | 17.0|